File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/mesh/persistence/androidxml/XmlPosition.java
package com.abubusoft.xenon.mesh.persistence.androidxml;

import com.abubusoft.kripton.annotation.Bind;
import com.abubusoft.kripton.annotation.BindType;
import com.abubusoft.kripton.annotation.BindXml;
import com.abubusoft.kripton.xml.XmlType;


@BindType
public class XmlPosition {

	@Bind
	@BindXml(xmlType=XmlType.ATTRIBUTE)
	public float x;
	
	@Bind
	@BindXml(xmlType=XmlType.ATTRIBUTE)
	public float y;
	
	@Bind
	@BindXml(xmlType=XmlType.ATTRIBUTE)
	public float z;
}

File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/mesh/persistence/androidxml/XmlPosition.kt
package com.abubusoft.xenon.mesh.persistence.androidxml

import com.abubusoft.kripton.annotation.Bind
import com.abubusoft.kripton.annotation.BindType
import com.abubusoft.kripton.annotation.BindXml
import com.abubusoft.kripton.xml.XmlType

@BindType
class XmlPosition {
    @Bind
    @BindXml(xmlType = XmlType.ATTRIBUTE)
    var x = 0f

    @Bind
    @BindXml(xmlType = XmlType.ATTRIBUTE)
    var y = 0f

    @Bind
    @BindXml(xmlType = XmlType.ATTRIBUTE)
    var z = 0f
}